algorithms/benches/snark/gm17.rs

```diff
@@ -29,7 +29,7 @@ use rand::{
     {self},
 };
 
-type GM17SNARK = GM17<Bls12_377, Benchmark<Fr>, Fr>;
+type GM17SNARK = GM17<Bls12_377, Fr>;
```

Comment:
Explain: This is an example that the "circuit" is dropped from the SNARK implementations.